webapp-django/crashstats/settings/base.py

```diff
@@ -451,9 +449,7 @@ def filter(self, record):
 
 CACHES = {
     "default": {
-        "BACKEND": config(
-            "CACHE_BACKEND", "django.core.cache.backends.memcached.MemcachedCache"
-        ),
```

Comment:
This used to be configurable because (I think) we set it to LocMemCache in the tests. We don't do that anymore, so I made it unconfigurable.